applications/crossbar/src/crossbar_doc.erl

```diff
@@ -133,11 +133,11 @@ load(DocId, Context, Options) ->
 
 load(_DocId, Context, _Options, 'error') -> Context;
 load(DocId, Context, Opts, _RespStatus) when is_binary(DocId) ->
-    OpenFun = case props:get_is_true('use_cache', Opts, 'true') of
-                  'true' -> fun couch_mgr:open_cache_doc/3;
-                  'false' -> fun couch_mgr:open_doc/3
-              end,
-    case OpenFun(cb_context:account_db(Context), DocId, Opts) of
+    Opener = case props:get_is_true('use_cache', Opts, 'true') of
+                 'true' ->  'open_cache_doc';
+                 'false' -> 'open_doc'
+             end,
+    case couch_mgr:Opener(cb_context:account_db(Context), DocId, Opts) of
```

Comment:
I don't understand the purpose of this change; was the previous code not working properly? Did you measure execution time and see a measurable improvement? If so, please share your tests; otherwise revert this chunk as it isn't related to the ticket.